css/properties/border-image-width.json

```diff
@@ -50,7 +50,7 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "18"
```

Comment:
Yes, I'm certain -- I used the collector and tested each version of Chrome to track down the exact version number!

Reply:
The following evaluates to `false` in Chrome 15, and to `true` in Chrome 18:

```js
'border-image-width' in document.createElement('div').style
```

This indicates to me that either `border-image-width` was only supported from Chrome 18.

Reply:
Chrome 16 starts supporting the property, Chrome 18 the `auto` value:

<img width="627" alt="image" src="https://github.com/user-attachments/assets/256cc304-6486-437c-b087-ba771325f05a">

Reply:
And the collector test fails in Chrome 16, because `div.style['border-image-width']` is `"auto auto auto auto"` when setting it to `"auto"`. I guess this could be worth a note about Chrome expanding the short-hand in Chrome 16/17, but it looks like Chrome 16 actually supports this value.

PS: Have you considered capturing some context (here: the actual value, in other cases: the error that calling e.g. a method throws) in the collector test results?